src/app/[locale]/map-center/page.tsx

```diff
@@ -1,14 +1,20 @@
+'use client';
+
 import dynamic from 'next/dynamic';
+import { useSearchParams } from 'next/navigation';
 
 const MapCenterPage = () => {
+  const searchParams = useSearchParams();
```

Comment:
Aren't you passing the 'search' | 'sell' entryType in the request body rather than in the query parameters?  
It looks like the coordinates are being distinguished and stored correctly between **search_map_center** and **sell_map_center**, but there seems to be a disconnect when retrieving them.  Could you take a closer look at this when you have a moment, @DarinHajou?  Feel free to ping me if you have any questions.

Reply:
> Aren't you passing the 'search' | 'sell' entryType in the request body rather than in the query parameters? It looks like the coordinates are being distinguished and stored correctly between **search_map_center** and **sell_map_center**, but there seems to be a disconnect when retrieving them. Could you take a closer look at this when you have a moment, @DarinHajou? Feel free to ping me if you have any questions.

Nevermind, @DarinHajou.  After looking into it further, it turns out that the **'search'** entryType is being passed and retrieved correctly, but there seems to be an issue with the coordinates, which is preventing the map marker from being rendered at the saved location.  For the Sell Center, the entryType is being passed and retrieved as 'null.'  It looks like the backend logic is accepting it as a sell type due to the conditional `if (entryType === 'search' ? search_map_center : sell_map_center)` and is working in your favor by saving to the sell_map_center coordinates by chance.  The map marker isn't rendering at the saved location though.  Hope this makes sense.  Feel free to reach out otherwise.